Domain/Models/Partner.cs

```diff
@@ -0,0 +1,22 @@
+﻿using MongoDB.Bson.Serialization.Attributes;
+
+namespace Samauma.Domain.Models
+{
+    public class Partner : Model
```

Comment:
Usar a notação para ignorar elementos extras nas models, vide link

https://github.com/the-web-forest/Ipe/blob/4a4db7508d83961f293d606f489bd42409230b52/Domain/Models/Order.cs#L5

**[BsonIgnoreExtraElements]**